packages/tradershub/src/components/RegulationSwitcher/RegulationSwitcherDesktop.tsx

```diff
@@ -45,7 +44,7 @@ const RegulationSwitcherDesktop = () => {
     };
 
     useEffect(() => {
-        if (activeTrading?.loginid.startsWith('CR') || isDIEL || isHighRisk) {
```

Comment:
Is removal of isDIEL temporary ?

Reply:
No. This condition is always true for DIEL countries. The 2 rest checks should do the trick